### PR TITLE
Add ClickHouse docs

### PR DESCRIPTION
**Pull request by BentsiLeviav**

### CODE REVIEW

**Comment by BentsiLeviav** [comment]
@microsoft-github-policy-service agree company="ClickHouse"

**Comment by Court72** [comment]
@BentsiLeviav, Thanks for your pull request. A member of the content team will review it.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: powerquery-docs/connectors/clickhouse.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+> [!NOTE]
+> Some capabilities may be present in one product but not others due to deployment schedules and host-specific
+> capabilities.
+
+## Prerequisites
+
+* A running ClickHouse setup
+* Customers must install the [ClickHouse ODBC driver](https://github.com/ClickHouse/clickhouse-odbc) before using the
+  connector
```
Comment (mshustov) [inline powerquery-docs/connectors/clickhouse.md:34]: Do we need a link to the docs on the official website? https://clickhouse.com/docs/en/integrations/powerbi For example, the doc doesn't provide details on where to get user credentials on ClickHouse Cloud

Comment (BentsiLeviav) [reply powerquery-docs/connectors/clickhouse.md:34]: Added "More Resources" section.

**Comment by BentsiLeviav** [comment]
Hi @Court72 
Has someone from the content team had a chance to review it?

**Comment by Court72** [comment]
@BentsiLeviav - It looks like @DougKlopfenstein added the `waiting-for-feature-release` label 3 weeks ago.

**Comment by DougKlopfenstein** [comment]
@BentsiLeviav - I've edited the text of the article for spelling, grammar, formatting, and Microsoft style. However, there are a couple of images that need to be updated before the article can be published:
* powerbi_05.png - shows how to get data from a screen that I believe no longer exists in the current version of Power BI Desktop. Could you update the image (and possibly update the step that describes the image)?
* powerbi_09.png - includes blurred out information in the navigator. We no longer publish images with blurred out information because newer AI models can unblur images and reveal the information below. Either provide an image with the area with blurred information showing only the background color, or provide fake information as described [here](https://review.learn.microsoft.com/en-us/help/contribute/contribute-legal-guidelines?branch=main).

In addition, several images (powerbi_06.png, powerbi_07.png, and powerbi_08.png) have different names for your connector (ClickHouseConnector and ClickHouseODBCConnector). If those images are obsolete (that is, your connector now has a consistent name throughout the screenshots), please update those images as well.

**Comment by BentsiLeviav** [comment]
Hi @DougKlopfenstein 

We missed your comments. As required, I've updated all screenshots to use the latest version of PowerBI. 
In addition, I didn't use any blurring as required.

**Review by DougKlopfenstein — APPROVED** [review]

**Pull request merged**